README.md

```diff
@@ -1,39 +1,166 @@
+We built a full stack - MERN Stack media streaming site with Stripe for Payments, and deployed it to Heroku at https://fantinder.herokuapp.com/ This project has played a key role in our journey to becoming a full-stack web developers. It showcases collaborative skills and coding abilities, especially in the context of a scalable, user-focused MERN app.
+Project Requirements were to to create a MERN stack single-page application that works with real-world data to solve a real-world challenge, with a focus on user demand:
+- Using React for the front end.
+- Using GraphQL with a Node.js and Express.js server.
+- Using MongoDB and the Mongoose ODM for the database.
+- Using queries and mutations for retrieving, adding, updating, and deleting data.
+- Deployed using Heroku (with data).
+- Meets the minimum requirements of a PWA: Uses a web manifest, Uses a service worker for offline functionality, Is installable
+- Offering a polished UI, is responsive and is interactive 
+- Including authentication (JWT) and protects sensitive API key information on the server.
+- The project has a clean repository that meets quality coding standards (file structure, naming conventions, best practices for class and id naming conventions, indentation, quality comments, etc.) and a high-quality README (with unique name, description, technologies used, screenshot, and link to deployed application).
+- This project is part of our REACT portfolio 
+- Improved CSS Styling and using a component library
+- Integrates a Payment Platform integrating the Stripe payment .
 
+## Contents
+- [Description](#Description)
+- [App Screenshot](#App)
+- [User Story](#User-Story)
+- [Acceptance Criteria](#Acceptance-Criteria)
+- [Concepts](#Concepts)
+- [License](#License)
+- [Author](#Author)
 
-### Application Screenshot
+## Description
+We are in a time where travel options are limited and a lot of people are spending time at home and watching movies together. Selecting a movie to watch together is always challenge. This is a application that makes that process simpler
+For MVP we assume that application users will communicate with each other outside the application only
+Additional potential features we considered
+….Feature 1: The movies in personalized profile favorites will show the other users (small bubble) who also have the same movie saved. 
+….Feature 2: On clicking on the bubble of a user who is interested in the same content as you, you will see more information (profile information) about them
+….Feature 3: As opposed to contact the user externally you can contact another use in app, via an in app chat
+….Feature 4: Not all users are friends, and users profile favorite content will be compared to their pre-approved friends only
 
+### App Screenshot
+![](images/FANTINDER.png)
 
-### User Story
+
+## User Story
 
 ```text
 AS A movie-lover stuck at home, 
 I WANT to search for & save movies,
-purchace movie/s using stripe payments,
+purchase movie/s using stripe payments,
```

Comment:
This is nitpicky, but could we revise this to a more general "what" phrase rather than a "how" phrase? I think specifying stripe here is too much detail for a user story.

For example: 
```
AS A movie-lover stuck at home,
I WANT to
- discover new movies
- save movies I want to watch later
- purchase and stream movies from home
```

Reply:
done